packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -3,18 +3,16 @@ import { BoosterConfig, EventEnvelope, Logger, UUID } from '@boostercloud/framew
 import { DynamoDB } from 'aws-sdk'
 import { eventsStoreAttributes } from '../constants'
 import { partitionKeyForEvent } from './partition-keys'
-import { Converter } from 'aws-sdk/clients/dynamodb'
+import { Converter, DocumentClient } from 'aws-sdk/clients/dynamodb'
 
 // eslint-disable-next-line @typescript-eslint/no-magic-numbers
 const originOfTime = new Date(0).toISOString()
 
 export function rawEventsToEnvelopes(rawEvents: DynamoDBStreamEvent): Array<EventEnvelope> {
-  return rawEvents.Records.map(
+  return rawEvents.Records.filter((item: DynamoDBRecord) => item.dynamodb?.NewImage).map(
```

Comment:
There is a reason for this change and that's because now we are removing items from the dynamo db events table, we'll receive that as a change to be published in the dynamodb stream. The deleted items doesn't contain the `NewImage` field so instead of throwing an exception and avoid processing the stream, here simply we'll get only the items with `NewImage`.

Also we don't need to process deleted events in the stream since we've a way to register explicitly a new deleted event on the command side and it will be processed as always.